Druid-Tests/DruidTestRTLCompiler.class.st

```diff
@@ -3254,10 +3254,11 @@ DruidTestRTLCompiler >> gen_pushLiteralVariable16CasesBytecode [
 DruidTestRTLCompiler >> gen_pushReceiverBytecode [
 	"AutoGenerated by Druid"
 
-	| currentBlock |
+	| R6 currentBlock |1 halt.
```

Comment:
Halt!

Anyway, this method is generated.

Reply:
Yes, I'll check, running the test should override it

Reply:
Fix coming